src/components/dialogs/script-dialog.js

```diff
@@ -51,8 +54,9 @@ const useStyles = makeStyles(() => ({
  * @param onError handle errors
  * @param title Title of the dialog
  * @param type Contingencies or filter
+ * @param subtype Element's subtype
  */
-const ScriptDialog = ({ id, open, onClose, onError, title, type }) => {
+const ScriptDialog = ({ id, open, onClose, onError, title, type, subtype }) => {
```

Comment:
I don't thnik you need a subtype prop here, the main type is sufficient to manage conditions in this dialog.

Reply:
done